source/includes/app-publishers.md

```diff
@@ -3,7 +3,7 @@
 > Don't forget to replace `{API_KEY}` with your actual API key.
 
 ```shell
-curl --compress -u "{API_KEY}:X" "https://api.appmonsta.com/v1/stores/android/publishers.json"
+curl --compress -u "{API_KEY}:X" "https://api.appmonsta.com/v1/stores/android/publishers.json?date={{ date }}"
```

Comment:
This one is optional, right?

Reply:
Err, is `date` a valid parameter for this call?

Reply:
Also there's a code typo (`$date`) in Java code:

```
 .queryString("date", $date)
```

Let's make sure we actually run code samples, not just fix them.

Reply:
`date` is valid and required for app publishers api call:

```
$ curl --compressed -u "cata250907e3cd959e1cf0ea07beb3a3ee5f1348:X" "https://api.appmonsta.com/v1/stores/android/publishers.json"
{"message": "date is a required query parameter."}
```

Reply:
Will be going through all the code samples and make sure they work!

Reply:
https://github.com/AppMonsta/slate/commit/6a6e795fd366016d1a09c83c5dec646f981324c3